lib/awful/widget/calendar_popup.lua

```diff
@@ -272,6 +273,7 @@ end
 -- @tparam string args.bg Wibox background color
 -- @tparam string args.font Calendar font
 -- @tparam number args.spacing Calendar spacing
+-- @tparam number args.margin Margin around calendar widget
```

Comment:
maybe it can be checked in a similar way like luacheck, or mb even there is a way to plug external checks there